home-automation/02-device-registry/README.md

```diff
@@ -6,8 +6,8 @@ All responses will have the form
 
 ```json
 {
-    "data": "Mixed type holding the content of the response",
     "message": "Description of what happened"
```

Comment:
JSON data does not have guaranteed order for a keys

Also the comma is on the wrong one so the syntax is invalid now.